invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/help/search.sv.html

```diff
@@ -0,0 +1,254 @@
+{#
+  Copyright (C) 2021 CERN.
+  Copyright (C) 2024 KTH Royal Institute of Technology.
+
+  Invenio App RDM is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
```

Comment:
Is this part to be in english regardless? It seems so to me since the german translation has this in english as well.

Reply:
Copyright notices should remain in English as they are typically standardized across all translations. Keeps it consistent with the MIT License wording.